Public/Src/Tools/SymbolDaemon/SymbolConfig.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Tool.SymbolDaemon
+{
+    /// <summary>
+    /// Symbol publishing request config.
+    /// </summary>
+    public sealed class SymbolConfig

```

Comment:
>SymbolConfig [](start = 24, length = 12)

I'm missing options to rewrite physical source location to source code url links...

* Is this not part of this api?
* Is there another tool / api comming?
* Do we have to use SourceLink? Is that compatible with caching?

Reply:
It's not a part of the API. The API only allows you to register the symbol data / upload a file. When I talked to Ben, he mentioned that 'links imbedding' is baked into the tools they run during the build, i.e., the binaries are ready for symbol upload once the appropriate pips are finished.

---
In reply to: [313488203](https://github.com/microsoft/BuildXL/pull/727#discussion_r313488203) [](ancestors = 313488203)